Formula/reorder-python-imports.rb

```diff
@@ -0,0 +1,33 @@
+class ReorderPythonImports < Formula
+  include Language::Python::Virtualenv
+
+  desc "Rewrites source to reorder python imports"
+  homepage "https://github.com/asottile/reorder_python_imports"
+  url "https://github.com/asottile/reorder_python_imports/archive/v2.3.0.tar.gz"
+  sha256 "c544b19a2900220e6a8d1a070dd651ce0c6885ca592f4e4af99f7551b6d9807b"
+
+  depends_on "python"
```

Comment:
Can this software work with Python 3.8? If so, please change this line to `depends_on "python@3.8`.

Reply:
I. think it still can be installed. Just wanted to make sure with asking you.

Reply:
That's right, yes. The `depends_on` line will install Python 3.8 alongside.

Reply:
`depends_on "python@3.8"` will indeed make it on 3.8 rather than 3.7.

However, this is the direction most formulae are taking.

At the time of writing:
Number of formulae which depend on Python 3.8: 359
Number of formulae which depend on Python 3.7: 37

We expect to bring the 3.7 number down to about 2-3 over the next couple weeks, at which point we will create `python@3.7` and `python` will become an alias to 3.8 (and 3.8 will no longer be keg-only). No formulae will depend on a generic `python` at that point and going forward we continue to use the approach of depending on a specific version of Python within formulae as it makes it much easier to add 3.9 when it arrives.